src/postgres/mod.rs

```diff
@@ -62,6 +67,17 @@ impl Postgres {
         self
     }
 }
+impl crate::InitSql for Postgres {
```

Comment:
this is essentially the same code for every intialisation.
I don't know if there is a way to move this to the trait.

Reply:
Let's drop the trait for now. Just in-place methods are ok. We can change in the future with fallback to trait